docs/pages/server-access/guides/ansible.mdx

```diff
@@ -62,7 +62,9 @@ ssh_args = -F ./ssh.cfg
 You can create an inventory file `hosts` manually or use a script below to generate it from your environment:
 
 ```code
-$ tsh ls --format=json | jq -r '.[].spec.hostname' > hosts
+# Set your cluster name (e.g. teleport.example.com or in the form mytenant.teleport.sh for Teleport Enterprise Cloud)
+$ CLUSTER=teleport.example.com
```

Comment:
Would it be possible to use the `Var` component instead of an env var here? I think that would improve the copy/paste experience

Reply:
Yes, please see update @ptgott